src/components/search/search.html

```diff
@@ -61,3 +61,25 @@ <h3 class="docs-block__header">Small</h3>
         </div>
     </div>
 </section>
+<section class="docs-block">
+    <aside class="docs-block__info">
+        <h3 class="docs-block__header">Small alt</h3>
+    </aside>
+    <div class="docs-block__content">
+        <div class="docs-block__contrast-box docs-block__contrast-box--full-width">
+            <div class="sg-search sg-search--alt">
+                <input type="search" placeholder="Find all the answers..."
+                       class="sg-input sg-input--light sg-input--full-width"/>
+                <div class="sg-search__button sg-search__button">
+                    <button class="sg-icon-as-button sg-icon-as-button--small" type="submit">
+                        <div class="sg-icon-as-button__hole">
+                            <svg class="sg-icon sg-icon--x18">
+                                <use xlink:href="#icon-search"></use>
+                            </svg>
+                        </div>
+                    </button>
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
```

Comment:
please add additional empty line at the end of the file - https://github.com/brainly/frontend-tools-configs/blob/master/.editorconfig#L5